### PR TITLE
Update to AWS SDK v1.11.95

### PR DESCRIPTION
**Pull request by rtyley**
Membership-Common is updating to AWS SDK v1.11.95, so subscriptions-frontend wants to stay in sync!

https://github.com/guardian/membership-common/pull/433

cc @AWare @svillafe

### CODE REVIEW

File: app/services/ExactTargetService.scala [hunk 1]
```diff
@@ -226,8 +226,10 @@ class ExactTargetService(
 }
 
 object SqsClient extends LazyLogging {
-  private val sqsClient = new AmazonSQSClient(CredentialsProvider)
-  sqsClient.setRegion(Region.getRegion(Regions.EU_WEST_1))
+  private val sqsClient = AmazonSQSClient.builder
```
Comment (johnduffell) [inline app/services/ExactTargetService.scala]: if it's chained like that I'd be tempted to put the `AmazonSQSClient.builder` on the next line just to be clearer

**Comment by johnduffell** [comment]
great, thanks! 👍 🚀

File: app/services/ExactTargetService.scala (hunk 1 above)
Comment (rtyley) [inline app/services/ExactTargetService.scala]: Yeah, I haven't quite settled on the best style for this builder stuff yet... think I'll go with this for now.

Comment (johnduffell) [inline app/services/ExactTargetService.scala]: yeah I always struggle with formatting for chained stuff - usually for me the answer is "don't" but I can see why in this case it's not meaningful to split things in any other way.

**Comment by rtyley** [comment]
Prout has failed to put acceptance test results on this PR (possibly due to an issue related to https://github.com/guardian/prout/issues/43 ) - but several acceptance tests have passed since this was merged, and I can still get to https://subscribe.theguardian.com/test-users (uses AWS S3) and CloudWatch custom metrics are [still getting written to](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#metricsV2:graph=~(view~'timeSeries~stacked~false~metrics~(~(~'subscriptions-frontend~'2XX-response-code~'Services~'Identity~'Stage~'PROD~'ResponseMethod~'POST~(stat~'Sum)))~region~'eu-west-1~start~'-PT3H~end~'P0D);namespace=subscriptions-frontend;dimensions=ResponseMethod,Services,Stage), so I'd say this change is good.

**Pull request merged**